web/src/components/molecules/Content/ContentDetailsMolecule/index.tsx

```diff
@@ -0,0 +1,64 @@
+import styled from "@emotion/styled";
```

Comment:
Please rename this component's folder to ContentDetails. its a child of the molecules directory so we don't need this to be ContentDetailsMolecule

Reply:
Renamed the folder.

Reply:
Sorry my bad, if you can take it one step further and rename the inner folders, removing the Content since they are already in the content folder...

Reply:
Done, renamed all the folders.